cmd/kube-proxy/app/conntrack.go

```diff
@@ -46,3 +62,16 @@ func (realConntracker) SetTCPEstablishedTimeout(seconds int) error {
 	glog.Infof("Setting nf_conntrack_tcp_timeout_established to %d", seconds)
 	return sysctl.SetSysctl("net/netfilter/nf_conntrack_tcp_timeout_established", seconds)
 }
+
+// isSysFSWritable checks /proc/mounts to see whether sysfs is 'rw' or not.
+func isSysFSWritable() (bool, error) {
+	const (
+		permScript   = "cat /proc/mounts | grep sysfs | awk -F\"[@ ,]\" '{printf $4}'"
+		permWritable = "rw"
+	)
+	perm, err := exec.Command("sh", "-c", permScript).CombinedOutput()
```

Comment:
Dependency on multiple commands. Is it possible to re purpose the Mounter.List/listProcMounts code to do this? https://github.com/kubernetes/kubernetes/blob/f2ddd60eb9e7e9e29f7a105a9a8fa020042e8e52/pkg/util/mount/mount_linux.go#L174 (Was that already ruled out for some other reason?)